### PR TITLE
Fix mobile wallet assets loading

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: packages/shared/src/store/assets/fungibleTokensSlice.ts [hunk 1]
```diff
@@ -46,6 +47,12 @@ const fungibleTokensSlice = createSlice({
       .addCase(syncVerifiedFungibleTokens.pending, (state) => {
         state.loadingVerified = true
       })
+      .addCase(syncFungibleTokensInfo.pending, (state) => {
+        state.loadingUnverified = true
+      })
+      .addCase(syncUnknownTokensInfo.pending, (state) => {
```
Comment (mvaivre) [inline packages/shared/src/store/assets/fungibleTokensSlice.ts:53]: Nit: Should we rename `syncUnknownTokensInfo` into something like `syncTokenTypes` to better match the `loadingTokenTypes` bool?

**Review by mvaivre — APPROVED** [review]
One tiny nit suggestion, but LGTM!

File: apps/mobile-wallet/src/components/AddressesTokensList.tsx [hunk 2]
```diff
@@ -152,6 +154,12 @@ const AddressesTokensList = ({ addressHash, isRefreshing, style }: AddressesToke
   )
 }
 
+const AssetNumberLoader = () => (
```
Comment (mvaivre) [inline apps/mobile-wallet/src/components/AddressesTokensList.tsx:157]: FYI: Added a spinner in the asset tabs when loading.

File: packages/shared/src/store/assets/fungibleTokensSlice.ts (hunk 1 above)
Comment (nop33) [reply packages/shared/src/store/assets/fungibleTokensSlice.ts:53]: `syncUnknownTokensInfo` is doing 3 things:
1. Gets the types
2. Calls `syncFungibleTokensInfo`
3. Calls `syncNFTsInfo`

**Pull request merged**